src/router/index.js

```diff
@@ -132,7 +132,7 @@ function configRoutes() {
         },
         {
           path: 'projects/:uuid/findings/:vulnerability',
-          name: 'Project Finding Lookup',
+          name: 'Project Vulnerability Lookup',
```

Comment:
I noticed this existing name clash because vue-router was complaining about it in the console.